sway-lib-std/src/storage.sw

```diff
@@ -1,6 +1,9 @@
 library r#storage;
 
+use ::assert::assert;
```

Comment:
Alphabetize the imports :) I know they weren't alphabetized before but might as well do it now.